crane/cli.py

```diff
@@ -38,7 +45,7 @@ def strip_trailing_slash(_, param, value):
 @click.option('--sleep-after-upgrade', envvar='CRANE_SLEEP_AFTER_UPGRADE', default=0, help='seconds to wait after upgrade', show_default=True)
 @click.option('--manual-finish', envvar='CRANE_MANUAL_FINISH', default=False, is_flag=True, help='skip automatic upgrade finish')
 @click.option('--slack-token', envvar='CRANE_SLACK_TOKEN', default=None, help='Slack API token')
-@click.option('--slack-channel', envvar='CRANE_SLACK_CHANNEL', default=None, help='Slack channel to announce in')
+@click.option('--slack-channels', envvar='CRANE_SLACK_CHANNELS', default=None, help='Slack channels to announce in', callback=split_to_list)
```

Comment:
Or do we want to keep the name "--slack-channel" and "CRANE_SLACK_CHANNEL"?

Reply:
You can see how we do this for `--service` and `--webhook-url`, which already take multiple values: we use https://click.palletsprojects.com/en/7.x/options/#multiple-options